docs/contributing/code-conventions.stories.mdx

```diff
@@ -0,0 +1,40 @@
+import { Meta } from '../../.storybook/components';
+
+<Meta title="Introduction/Contributing/Code Conventions" />
+
+# Code Conventions
+
+When developing components in Circuit UI, we have a few development principles you should keep in mind. They will help you with picking good names, design clear & understandable APIs, and keep things consistent.
+
+## Best Practices
+
+**Every component must be thoroughly tested.** If the component is purely representative (e.g. simple, styled components), a snapshot test is sufficient. If the component is interactive, you should do your best to test the interaction on a unit level.
+
+**Ensure accessibility and autocompletion.** Every component should have a minimum of an accessibility test asserted with `jest-axe`. You should manually check mouse, touch, and keyboard accessibility as well as good screen reader support. You should also make a special effort to properly enable autocompletion for relevant fields, using the proper input names and autocomplete properties.
+
+**All translated text should be provided as props with English defaults.** It must be possible for all user-facing text inside a component to be replaced with translated versions. This also makes it easier to stick to content guidelines by providing a good starting example.
+
+**Documentation is part of your component.** Be sure to write meaningful documentation about best practices / usage guidelines when creating or updating existing components. You can use existing documentation for reference.
+
+## Component Design Principles
```

Comment:
Would be cool to add more principles here in the future 🙂

Reply:
Absolutely. I just moved this content around. Would love to extend it.